CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ UNRELEASED
 * [ [#1345](https://github.com/digitalfabrik/integreat-cms/issues/1345) ] Track DeepL API usage by regions
 * [ [#1695](https://github.com/digitalfabrik/integreat-cms/issues/1695) ] Treat URLs with broken hash anchors as valid in link checker
 * [ [#1969](https://github.com/digitalfabrik/integreat-cms/issues/1969) ] Hide edit link button in content forms for users without edit permission
+* [ [#1954](https://github.com/digitalfabrik/integreat-cms/issues/1954) ] Fix RTL text direction in PDF export
```

Comment:
Due to the long time this PR is open now, this needs to be rebased and then moved up into the unreleased section again.